src/components/AdForm/AdForm.styles.js

```diff
@@ -54,24 +54,24 @@ export const Form = styled.form`
   }
 
   a {
-    color: #3E9EC8;
+    color: #3e9ec8;
     padding: 10px;
     border-radius: 18px;
-    font: 500 12px 'Rubik', sans-serif;
+    font: 500 12px "Rubik", sans-serif;
     width: 90%;
     text-align: center;
     cursor: pointer;
     text-decoration: none;
   }
 
   input {
-    height: 40px;
-    border-radius:18px;
+    height: 35px;
+    border-radius: 18px;
     padding: 0 20px;
     color: #777;
     font-size: 15px;
     width: 90%;
-    border: 2px solid #F0D4A3
+    border: 2px solid #f0d4a3;
```

Comment:
Quando for trabalhar com cores, acho mais interessante trabalharmos com as variáveis CSS. Caso não haja uma cor que vc quer usar é só criar uma nova variável ;)